lib/Connection.js

```diff
@@ -1808,8 +1822,15 @@ function validateUIDList(uids, noThrow) {
         return err;
       else
         throw err;
-    } else if (typeof uids[i] !== 'number')
+    } else if (intval <= 0) {
```

Comment:
I don't know that we can include this part of the code, unfortunately.  The IMAP spec is clear that UIDs must be positive integers, but we have come across servers that use UIDs equal to `0`